libs/wire-subsystems/test/unit/Wire/UserStoreSpec.hs

```diff
@@ -33,3 +37,21 @@ spec = do
        in if (isJust storedUser.language)
             then user.userLocale === Locale (fromJust storedUser.language) storedUser.country
             else user.userLocale === defaultLocale
+
+  describe "UserStore effect" $ do
```

Comment:
I'm wondering if it wouldn't be useful to also test what happens when the user does not exist. Or, can't this case not happen? :thinking:

Reply:
Ah, I see. Then, I'm approving. :+1: